MotionMark/unit-tests/test-spiral-iterator.html

```diff
@@ -0,0 +1,58 @@
+<!--
```

Comment:
I don't think this is necessary unless SpiralIterator gets used in more than Multiply.

Reply:
The whole directory for unit test is removed.